src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,92 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final int NAME_INDEX_ZERO = 0;
+    public static final int NAME_INDEX_FIRST = 1;
+    public static final int NAME_INDEX_SECOND = 2;
+    public static final int THIRD = 3;
+    public static final int FOURTH = 4;
```

Comment:
constants name are not descriptive enough, not sure what that means just by looking at them